content/en/docs/storage-providers/seal-workers.md

```diff
@@ -204,73 +204,100 @@ Worker 1, host othercomputer
 
-Device numbers can be obtained with the `nvidia-smi -L` command.
+## Running Multiple Lotus Workers
 
-#### Sector Storage Groups
+Storage providers intending to scale significantly beyond the 10TB minimum will likely want to run multiple Lotus Workers, each on a dedicated machine. Multi-worker environments can benefit from additional configuration. Each case will have unique requirements and considerations, and this document can provide only a very brief overview of some options.
 
-The sectorstore.json contains two additional optional fields to allow for creating worker groups and avoiding unnecessarily moving data between multi-purpose workers.
-
-##### Use case
-
-- This feature is useful when the Lotus storage path is not shared between workers.
-- This feature can be used to group workers together if some, but not all, of them share a storage path (e.g. NFS). If all the workers share the same storage path, then this feature should not be used.
-- This feature does not relate to long term storage path.
+### Sector Storage Groups
 
+As of Lotus v1.13.2, the `sectorstore.json` file in each storage location contains two additional, optional fields to allow for creating worker groups to avoid unnecessarily moving data between multi-purpose workers.
 ```
 Groups []string - list of group names the storage path belongs to.
 AllowTo []string - list of group names to which sectors can be fetched to from this storage path.
 ```
+ The `Groups` field identifies the storage path as one that accepts incoming PC1 sectors for PC2 processing. `AllowTo` determines which storage path(s) a sector may be moved to for PC2. **If `AllowTo` is not specified, the store will be accessible to all groups.**
+ 
+- This feature is useful when sealing storage paths are not shared among workers.
+- This feature can be used to group workers together if some, but not all, of them share a storage path (e.g., NFS). If all workers share the same storage path, then this feature should not be used.
+- This feature does not relate to the long term storage path.
 
-The option `AllowTo` defined which storage path the sector can be pulled from.
-For example in the following setup:
+#### Use case
 
+Consider a setup with three workers: Worker 1 (PC1, PC2); Worker 2 (PC1, PC2); Worker 3 (PC1). Without storage groups, PC2 tasks on Workers 1 or 2 could be scheduled with sector data from any of the three workers. For example, if Worker 1 finished a PC1 job but did not yet have an available PC2 window, the large volume of data generated in PC1 might be needlessly moved to Worker 2 for PC2. This wastes bandwidth and can cause data fetching to block data processing. The following `sectorstore.json` files are configured to avoid such unnecessary file transfers.
+- Worker 1 (PC1, PC2):
```

Comment:
```suggestion
Consider a setup with three workers: 

- Worker 1 (PC1, PC2)
- Worker 2 (PC1, PC2)
- Worker 3 (PC1). 

Without storage groups, PC2 tasks on Workers 1 or 2 could be scheduled with sector data from any of the three workers. For example, if Worker 1 finished a PC1 job but did not yet have an available PC2 window, the large volume of data generated in PC1 might be needlessly moved to Worker 2 for PC2. This wastes bandwidth and can cause data fetching to block data processing. The following `sectorstore.json` files are configured to avoid such unnecessary file transfers.

- Worker 1 (PC1, PC2):
```